### PR TITLE
EZEE-3224: Added ezplatform-segmentation package

### PR DESCRIPTION
**Pull request by webhdx**
> JIRA: https://jira.ez.no/browse/EZEE-3224

### CODE REVIEW

File: config/routes/ibexa_segmentation.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ibexa.segmentation:
```
Comment (ViniTou) [inline config/routes/ibexa_segmentation.yaml:1]:
```suggestion
ibexa.ezplatform.segmentation:
```
?

**Review by ViniTou — APPROVED** [review]

**Review by Nattfarinn — APPROVED** [review]

File: config/routes/ibexa_segmentation.yaml (hunk 1 above)
Comment (Nattfarinn) [reply config/routes/ibexa_segmentation.yaml:1]: @ViniTou Are you sure `ibexa.ezplatform` and not `ibexa.platform`?

Comment (lserwatka) [reply config/routes/ibexa_segmentation.yaml:1]: ibexa.platform

Comment (webhdx) [reply config/routes/ibexa_segmentation.yaml:1]: I see it has been already done for DAM Connector: https://github.com/ezsystems/ezplatform-ee/blob/master/config/routes/ezplatform_connector_dam.yaml

I also think `ibexa.platform.segmentation` is more appropriate, along with `ibexa_platform_segmentation.yaml` filename. We should fix DAM connector alias and filename to follow this notation.

Comment (webhdx) [reply config/routes/ibexa_segmentation.yaml]: Fixed in 4a63b183

**Review by adamwojs — APPROVED** [review]

**Review by lserwatka — APPROVED** [review]

**Comment by lserwatka** [comment]
You can merge it sideways

File: config/routes/ibexa_segmentation.yaml (hunk 1 above)
Comment (ViniTou) [reply config/routes/ibexa_segmentation.yaml]: Yes, my bad.
fix for DAM:
https://github.com/ezsystems/ezplatform-ee/pull/158

**Comment by webhdx** [comment]
@lserwatka Done.

**Pull request merged**